Grbl_Esp32/src/NutsBolts.cpp

```diff
@@ -231,3 +231,11 @@ char* trim(char* str) {
     end[1] = '\0';
     return str;
 }
+
+// Returns the number of set number of set bits
+// Would return 3 for 01100010
+int numberOfSetBits(uint32_t i) {
```

Comment:
How about using the GCC intrinsic __builtin_popcount(i);
```
int numberOfSetBits(uint32_t i) {
     return __builtin_popcount(a);
}
```

Reply:
OK, Changed...